### PR TITLE
SonarCloud Maven inline usage with JaCoCo XML report generation

### PR DESCRIPTION
**Pull request by axel3rd**
Defining JaCoCo plugin in Maven POM is not the easiest way when you are using multiple JDKs in your Travis build.

Proposal for inline commands, with focus on JaCoCo XML report generation.

### CODE REVIEW

**Comment by axel3rd** [comment]
@bellingard : Proposal as reviewer.

**Comment by axel3rd** [comment]
@MariadeAnton : Did you want a "local master fetch + squash + push force" on branch ? Or GitHub "Update branch" + squash during merge will be OK ?

**Comment by MariadeAnton** [comment]
Thanks so much for your PR, @axel3rd! Yes, GitHub Update branch + squash during merge works just fine :)

File: user/sonarcloud.md [hunk 1]
```diff
@@ -69,11 +69,20 @@ script:
 {: data-file=".travis.yml"}
 
 > Please note the following:
-- the "coverage" profile activates the generation of the JaCoCo XML report
+- the "coverage" profile (defined in your POM file) activates the generation of the JaCoCo XML report
 - "sonar.projectKey" can also be set as a property on the main POM file. Its value can be found on the right side of the project homepage on SonarCloud
 
 Please take a look at the [live Maven-based example project](https://github.com/SonarSource/sq-com_example_java-maven-travis) to know more about this use case.
 
+Without POM update, or if you are [Testing Against Multiple JDKs](languages/java/#testing-against-multiple-jdks) (SonarCloud analysis should be executed only once), or need multiple steps Maven commands ; execute [JaCoCo XML report generation](https://www.eclemma.org/jacoco/trunk/doc/report-mojo.html) at end of your main build. Script section would be like:
```
Comment (MariadeAnton) [inline user/sonarcloud.md]: Hey @axel3rd! 

Just to clarify a bit this paragraph, when you mention "without POM update", do you refer to defining the "coverage" profile in the POM file or something else?

Comment (axel3rd) [reply user/sonarcloud.md]: > do you refer to defining the "coverage" profile in the POM file or something else?

@MariadeAnton: Exactly !

=> Proposal:
```suggestion
Without POM update (e.g.: some profile defining *jacoco-maven-plugin* executions), or if you are [Testing Against Multiple JDKs](languages/java/#testing-against-multiple-jdks) (SonarCloud analysis should be executed only once), or need multiple steps Maven commands ; execute [JaCoCo XML report generation](https://www.eclemma.org/jacoco/trunk/doc/report-mojo.html) at end of your main build. Script section would be like:
```

Comment (MariadeAnton) [reply user/sonarcloud.md]: What about saying this directly? 

> If you don't have profile defining *jacoco-maven-plugin-executions*, or if .... 

Would that convey the right message? Or are there other POM updates that could affect this behaviour?

Comment (axel3rd) [reply user/sonarcloud.md]:
```suggestion
If you don't have POM profile defining *jacoco-maven-plugin* executions, or if you are [Testing Against Multiple JDKs](languages/java/#testing-against-multiple-jdks) (SonarCloud analysis should be executed only once), or need multiple steps Maven commands ; execute [JaCoCo XML report generation](https://www.eclemma.org/jacoco/trunk/doc/report-mojo.html) at end of your main build. Script section would be like:
```

Comment (axel3rd) [reply user/sonarcloud.md]: > Would that convey the right message? Or are there other POM updates that could affect this behaviour?

Agree with this more simple phrase. No others impacts affecting this behavior.

**Pull request merged**